tests/behaviour/expectations/behaviour.ts

```diff
@@ -25,4784 +25,4758 @@ export const expectations = flatten(
           Expect.Simple('test', [], []),
           Expect.Simple('returnTest', [], toCairoUint256(12)),
         ]),
```

Comment:
Let's remove the example test. There was a duplicate somewhere below so let's use the other one